source/user-manual/capabilities/file-integrity/fim-configuration.rst

```diff
@@ -80,26 +97,152 @@ This functionality uses Linux Audit subsystem and the Microsoft Windows SACL, so
+  - Size: 13
+  - Permissions: rw-r--r--
+  - Date: Wed Apr 1 16:30:28 2020
+  - Inode: 23188
+  - User: root (0)
+  - Group: root (0)
+  - MD5: ab68c0f2af74dc4a1c482a83c0c5a3ec
+  - SHA1: 3a686748bf79db0adbad15f22ef566212a13b6c8
+  - SHA256: 40e29c160ea4e9911cafb9bcdbb2bcec21904a0c13a2871936c79145ff8085c7What changed:
+  1c1
+  < Original text
+  ---
+  > Altered text
+
+If some sentive files exist in the monitored with ``report_changes`` path, :ref:`nodiff <reference_ossec_syscheck_nodiff>` option can be used. This option disables computing the diff for the listed files, avoiding data leaking by sending the files content changes through alerts:
+
+.. code-block:: xml
+
+ <syscheck>
+   <directories check_all="yes" realtime="yes" report_changes="yes">/test</directories>
+   <nodiff>/test/private</nodiff>
+ </syscheck>
+
+In this example, by adding :ref:`nodiff <reference_ossec_syscheck_nodiff>` option, syscheck will not compute the diff for a listed text file:
+
+.. code-block:: console
+
+  ** Alert 1585757658.154829: - ossec,syscheck,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,
+  2020 Apr 01 16:14:18 (agent) any->syscheck
+  Rule: 550 (level 7) -> 'Integrity checksum changed.'
+  File '/test/private' modified
+  Mode: real-time
+  Changed attributes: size,mtime,inode,md5,sha1,sha256
+  Size changed from '14' to '20'
+  Old modification time was: '1585757413', now it is '1585757658'
+  Old inode was: '23187', now it is '23185'
+  Old md5sum was: 'ef4ad1a40d0a95ad2e1b72eccdca6d44'
+  New md5sum is : '158ccd88359654ac4ffd0e3cecb79a49'
+  Old sha1sum was: '5f34d30f7bdefe9e825bff388de047dacdc09853'
+  New sha1sum is : '18dfef68273c00fc733e28ce9aa1830f5e8fabd8'
+  Old sha256sum was: '211ae95d4e54cff5724a98f0bae0b505adfdafe1ed8b15e40570a5fe58d20c61'
+  New sha256sum is : '60c2a08e66f02bacea882f7b437f9c983431d75a686b703661c34e288d36de9d'Attributes:
+  - Size: 20
+  - Permissions: rw-r--r--
+  - Date: Wed Apr 1 16:14:18 2020
+  - Inode: 23185
+  - User: root (0)
+  - Group: root (0)
+  - MD5: 158ccd88359654ac4ffd0e3cecb79a49
+  - SHA1: 18dfef68273c00fc733e28ce9aa1830f5e8fabd8
+  - SHA256: 60c2a08e66f02bacea882f7b437f9c983431d75a686b703661c34e288d36de9dWhat changed:
+  <Diff truncated because nodiff option>
+
 .. _how_to_fim_ignore:
 
-Configure to ignore files
--------------------------
+Configuring ignoring files and Windows registry entries
+-------------------------------------------------------
 
-Files and directories can be omitted using the ignore option (or registry_ignore for Windows registry entries). In order to avoid false positives, **syscheck** can be configured to ignore certain files that don't need to be monitored.
+In order to avoid false positives, syscheck can be configured to ignore certain files and directories that don’t need to be monitored by using the :ref:`ignore <reference_ossec_syscheck_ignore>` option:
```

Comment:
```suggestion
In order to avoid false positives, syscheck can be configured to ignore certain files and directories that do not need to be monitored by using the :ref:`ignore <reference_ossec_syscheck_ignore>` option:
```